desicochrane/basic-mailbox-api
# Add super admin

**🤗commit-by-commit-friendly🤗**

# Why does this PR exist?

We want to use basic authentication to protect our end points, this PR adds a super user to the system with an api token.

### How should this be manually tested?

1. copy the `.env.example` file to `.env`
1. open `.env` and set a `SUPER_ADMIN_EMAIL` to your email and `SUPER_ADMIN_TOKEN` to some random hash
1. run `php artisan migrate --seed` to create the table and seed your admin user
1. check the database, you should have a users table with a single user with the token and email that you set

### Discussion

This is not a robust or fully featured user system - just a minimal boilerplate to get us running with our prototype
